docs/data/toolpad/core/components/app-provider/AppProviderBasic.js

```diff
@@ -25,7 +25,11 @@ const NAVIGATION = [
   },
 ];
 
-const demoTheme = extendTheme({
+const demoTheme = createTheme({
+  cssVariables: {
+    colorSchemeSelector: 'data-toolpad-color-scheme',
```

Comment:
If this selector needs to be added every time I guess we could override it ourselves, or make it fully configurable. As right now I guess the `AppProvider` theming wouldn't work with any other selector.